frontend-ssr/components/notification-tooltip/Notification.module.scss

```diff
@@ -0,0 +1,135 @@
+@import "../../styles/variables";
+@import "../../styles/animations";
+
+.notifications-tooltip {
+  .icon .icon-spinner {
+    display: inline-block;
+    animation: rotateAnimation 1s linear infinite;
+  }
+
+  .icon .badge {
+    position: absolute;
+    top: 0;
+    transform: translateY(-50%);
+    right: -0.5em;
+    height: 1.5em;
+    min-width: 1.5em;
+    border-radius: 50%;
+    color: color(white);
+    background-color: color(red);
+    font-size: 0.8em;
+    line-height: 0.5em;
+    text-align: center;
+    padding: 0.5em 0 0 0.1em;
+  }
+
+  :global .scroll-wrapper {
+    position: absolute;
+    right: 0;
+    z-index: 2;
+    border: 1px solid #bcc2c6;
+    box-shadow: 0 0 1px #77838b;
+    background-color: color(white);
+    height: 15em;
+    width: 15em;
+    max-width: 100vw;
+  }
+
+  #markAllAsReadBtn {
+    border: 0;
+    background: none;
+    cursor: pointer;
+    padding: 0;
+    margin: 0.4em 0.75em 0 0;
+    display: inline-block;
+
+    &:hover {
+      color: color(blue);
+      font-weight: bold;
+    }
+  }
+
+  .notifications li {
+    font-size: 0.8em;
+    line-height: 2em;
+    border-bottom: 1px solid color(darker-white);
+    &:first-child {
+      text-align: right;
+      &:hover {
+        background-color: color(white);
+        background-image: none;
+        color: initial;
+      }
+    }
+    a {
+      display: flex;
+      padding: 0.5em 1em;
+      &:hover {
+        background-color: color(white);
+      }
+    }
+
+    &.is--unread {
+      background-color: darken(color(darker-white), 5%);
+    }
+
+    img {
+      width: 1.6em;
+      height: 1.6em;
+      margin-right: 0.8em;
+    }
+
+    .body {
+      white-space: initial;
+
+      .message {
+        margin: 0;
+      }
+
+      time {
+        color: color(silver);
+      }
+    }
+
+    .user {
+      border-bottom: 1px solid color(silver);
+      margin-right: 0.4em;
+    }
+  }
+
+  .hidden {
+    visibility: hidden;
+  }
+}
+
+.dummy-notification {
```

Comment:
Let's rename this to `skeleton` as in `ExerrciseModule.scss`. Also make sure there are as little duplicate styles as possible between the normal notification and the skeleton one.